Roadmap.md

```diff
@@ -0,0 +1,135 @@
+| [Mixed Reality Toolkit](#mixed-reality-toolkit-roadmap) | 2018.9.0.0 | September 2018 | [2018.9.0.0 (Version Next initial release)](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects/14) |
+| [Mixed Reality Toolkit](#mixed-toolkit-toolkit-roadmap) | 2018.10.0.0 | October 2018 | |
+| [Mixed Reality Toolkit](#mixed-toolkit-toolkit-roadmap) | 2018.11.0.0 | November 2018 | |
+
+Release details, including backlog items, can be found on the [GitHub project pages](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects).
+
+# Mixed Reality Toolkit (MRTK) Roadmap
+
+The Mixed Reality Toolkit (also known as "vNext") is an all-new product, built to be cross MR/AR/VR/XR platform by design. There are two planned pre-releases after which the Mixed Reality Toolkit will become the primary product.
+
+The Mixed Reality Toolkit will require Unity 2018 or newer.
+
+| When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will branch (ex: mrtk_2018_LTS) and require an LTS release. This branch will be targeted at developers who wish to have the most stable MRTK on which to build. It is expected that the branched Mixed Reality Toolkit will receive bug fixes and no new features. The mrtk_development branch will update to the next Unity Tech release (ex: 2019.1) and **may** drop backward compatibility.
+
+## New Architecture
+
+The MRTK is being rewritten to be cross AR/VR/MR/XR platform and to be more modular. The core layer will contain the interfaces and definitions required to build manager components (ex: input manager) as well as the abstractions needed to support multiple platforms. The SDK layer is where pre-built / reusable components will be delivered. The SDK will be further modularized to make it simple to opt-in to specific functionality (ex: UX controls).
+
+## July 2018 - Alpha
+
+The Mixed Reality Toolkit (MRTK) Alpha, released in late July 2018 is being released as a single package (core, sdk and examples) as the team introduces the new architecture. This release does **not** contain all of the features and functionality of the HoloToolkit releases.
+
+The theme(s) for the Alpha release are
+
+- Minimum Viable Product
+    - New architecture
+    - Render scene
+    - VR Boundary support
+    - Input and Interactions
+    - Locomotion / Teleportation
+    - Standard shader
+
+Supported platforms 
+
+- Windows Mixed Reality
+    - Microsoft HoloLens
+    - Immersive Headsets
+- OpenVR
+
+## August 2018 - Beta
+
+The MRTK Beta release will introduce initial support for modularization. It is also when the vast majority of HoloToolkit features will be added. Supported platforms will be unchanged from the Alpha release.
+
+| It is important to note that the goal is feature parity and **not** zero app code change. There will be consolidation of duplicated functionality as well as namespace, component and API changes.
```

Comment:
Is the "|" meant to be a ">" character to denote a blockquote?
Change or remove.

Reply:
It wasn't intended to be a blockquote, but I agree that is a better format for this.